### ISSUE TITLE
Not latest version from `apt` 

### ISSUE THREAD

**Issue by leobenkel** [usebruno/bruno#2841]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

I see that the latest version is something like `1.25` (https://github.com/usebruno/bruno/releases/tag/v1.25.0)  but when i try to update i get:

```
sudo apt install bruno
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
bruno is already the newest version (1.23.1).
0 upgraded, 0 newly installed, 0 to remove and 3 not upgraded.
```

why am i getting `1.23` instead of `1.25` ? 

### Screenshots/Live demo link

```
sudo apt install bruno
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
bruno is already the newest version (1.23.1).
0 upgraded, 0 newly installed, 0 to remove and 3 not upgraded.
```

**Comment by jwetzell** [comment]
@helloanoop

**Comment by leobenkel** [comment]
I think this is fixed now, thanks !

**Comment by abobov** [comment]
Currently, the latest version available is 1.27.0, although 1.28.0 came out a day ago.
